### PR TITLE
Add tracked entity type comment

### PR DESCRIPTION
**Pull request by kael89**
### Issue #:
[No issue]

### CODE REVIEW

**Review by billli0 — APPROVED** [review]
Looks good!

File: packages/database/src/modelClasses/Entity.js [hunk 1]
```diff
@@ -24,6 +24,8 @@ const WORLD = 'world';
 const PROJECT = 'project';
 const CITY = 'city';
 
+// Note: if a new type is not included in `ORG_UNIT_ENTITY_TYPES`,
```
Comment (IgorNadj) [inline packages/database/src/modelClasses/Entity.js:27]: Hi sorry what does this mean?

Comment (kael89) [reply packages/database/src/modelClasses/Entity.js:27]: I hoped that the comment would be self-explanatory... If it isn't, perhaps I need to change it 🙂

Comment (kael89) [reply packages/database/src/modelClasses/Entity.js:27]: It's related to

https://github.com/beyondessential/tupaia-backlog/issues/2115#issuecomment-780237062
https://github.com/beyondessential/tupaia-backlog/issues/2122#issuecomment-780191494

which potentially caused this issue yesterday: https://beyondessential.slack.com/archives/G01CW3WKFPS/p1613515567002300 (although it's hard to confirm)

Comment (IgorNadj) [reply packages/database/src/modelClasses/Entity.js:27]: Ah very useful info, thanks! As mentioned in chat maybe good to put a link in here to some docs about it

**Pull request merged**